### ISSUE TITLE
Bullet Journal support

### ISSUE THREAD

**Issue by danpker** [zweck/todo_cmd#8]
It would be cool to have some kind of bullet journal support. Using `>` `<` `*` etc.

**Comment by zweck** [comment]
Yeah, this doesn't seem too challenging. Just need to move the hacky regex that checks for `[ ]` to be part of a symbol library that does a check for a whole set of initial symbols and allows you to add your own.
